tasklist/els-schema/src/main/java/io/camunda/tasklist/util/IndexSchemaValidatorUtil.java

```diff
@@ -179,14 +174,37 @@ private IndexMappingDifference getIndexMappingDifference(
           // all those indices should have the same difference. Compare based only on the
           // differences (exclude the IndexMapping fields in the comparison)
         } else if (!difference.checkEqualityForDifferences(currentDifference)) {
-          throw new TasklistRuntimeException(
-              "Ambiguous schema update. First bring runtime and date indices to one schema. Difference 1: "
-                  + difference
-                  + ". Difference 2: "
-                  + currentDifference);
+          if (currentDifference.getEntriesDiffering().stream()
+              .anyMatch(this::nonDynamicPropertyDifference)) {
+            throw new TasklistRuntimeException(
+                "Ambiguous schema update. First bring runtime and data indices to one schema."
+                    + " Difference 1:"
+                    + difference
+                    + ". Difference 2: "
+                    + currentDifference);
+          } else {
+            LOGGER.debug(
+                String.format(
+                    "Difference is on dynamic field - continue initialization, left: %s, right: %s. Actual diff values: %s",
+                    difference.getLeftIndexMapping().getIndexName(),
+                    difference.getRightIndexMapping().getIndexName(),
+                    difference.getEntriesDiffering()));
+          }
         }
       }
     }
     return difference;
   }
+
+  private boolean nonDynamicPropertyDifference(final PropertyDifference propertyDifference) {
+    final Object typeDefinition = propertyDifference.getLeftValue().getTypeDefinition();
+    if (propertyDifference.getLeftValue().getTypeDefinition() instanceof Map) {
+      final Map<String, Object> typeDefMap = (Map<String, Object>) typeDefinition;
+      final Object dynamicValue = typeDefMap.getOrDefault("dynamic", false);
+      if (dynamicValue.equals("true")) {
```

Comment:
previously this was
```java
final Object dynamicValue = typeDefMap.getOrDefault("dynamic", false);
 if (dynamicValue.equals(true)) {
```
it is unclear to me whether the map contains strings or booleans or both ??

I propose we do the following

```suggestion
final boolean isDynamic = Boolean.parseBoolean(typeDefMap.getOrDefault("dynamic", "false").toString());
if (isDynamic) {
```

"true" → true
"false", null, and anything else → false
true (Boolean) → "true".toString() → true